en/key_concepts.rst

```diff
@@ -18,67 +21,98 @@ There are two types of nodes in Open Orchestra:
+See also `configuring a zone`_.
 
-See also `Presentation of the Content Types`_.
+Blocks
+~~~~~~
+
+A block is the base brick representing any visible element on a page.
+Aggregating blocks found in the nodes will lead to the construction of the pages.
+
+See also the `list of blocks`_ available.
+
+Transverses blocks
+~~~~~~~~~~~~~~~~~~
+
+Transverses blocks are defined inside the transverse node, they can be shared by several nodes of a website.
+All blocks can be transverse, as long as they are added to the transverse node.
+Transverse blocks can only be conigured in the transverse node.
+
+Templates
+~~~~~~~~~
+
+Templates are used to preconfigure nodes by setting zones that are commonly used accross pages.
+Open Orchestra let the user start with a template when creating a node.
+
+See also `configuring a template`_.
+
+Summary
+~~~~~~~
+
+This scheme represents how those different part are related to one another.
+
+.. image:: ../../images/drag_block.png
 
 Contents
 --------
 
-With Open Orchestra, contents are used to display contributed information (articles, news, etc.).
+Contents
+~~~~~~~~
+
+In Open Orchestra, contents are used to display contributed information (articles, news, etc.).
 The contents are defined by a content type.
 Like nodes, contents are multilingual, versionned and have a validation workflow.
 
 See also `Presentation of contents`_.
 
-Templates
----------
+Content types
+~~~~~~~~~~~~~
 
-Templates are used to preconfigure pages by setting zones that are commonly used accross pages.
-Open Orchestra let the user start with a template when creating a node.
+Content types are a way to create new kinds of content that will be available to contribution.
+Creating a content type allows to define the fields available in the future contents.
 
-See also `configuring a template`_.
+For instance, a 'news' content will have differents fields from an 'article' content,
+so there would be two distinct content types for these objects : one for the news contents
+and another for the article contents.
 
-Zones
------
+See also `Presentation of the Content Types`_.
 
-Zones are the way to organize the architecture of the page.
-They are embedded in the nodes and can contain other zones or blocks.
+Medias
+~~~~~~
 
-See also `configuring a zone`_.
+If you use the full Open Orchestra installation, you will be able to upload and managed different kind
+of medias.
 
-Blocks
-------
+The medias can be used in ``Contents`` or in ``Blocks``.
 
-A block is the base brick representing any visible element on a page.
-Aggregating blocks found in the nodes will lead to the construction of the pages.
+See also the `Media presentation`_.
 
-See also the `list of blocks`_ available.
+Keywords
+~~~~~~~~
 
-Transverses blocks
-------------------
+Keywords can be created in the Back Office, and then be used to tag contents and medias.
```

Comment:
In fact I have no idea of why keywords are used everywhere in the back office :s
I mean there's a utility to tag contents : we can create a tag cloud or get contents by tag, but I don't understand why content type are tagged for instance.
So if you also don't know what to tell about this, maybe this feature is useless and we should consider remove it where it is non required.